docs: add Monitor docs section (#79577)
metabase/metabase: docs/monitor/alerts-management.md

For each alert, Metabase shows the:

- **ID**: The alert's unique identifier.
- **Question**: The question that triggers the alert.
- **Owner**: The person who created the alert.
- **Channel**: Where the alert sends notifications (email, Slack, or webhook).
- **Last checked**: The timestamp for when Metabase last ran the alert's question.
- **Last send attempt**: The timestamp for when Metabase last tried to send the alert. Alerts that have never met their condition show **Never**.

If a check or send attempt failed, Metabase displays an error icon next to the timestamp.

To find a specific alert, search by question or owner. Click **Filter** to filter alerts by channel, last send attempt status, owner status, or email recipient.

## Delete alerts or change owner

To delete alerts or reassign them to a new owner:

1. Select the checkbox next to each alert. To select all alerts, select the checkbox in the header row.
2. In the action bar, click **Delete** or **Change owner**.
3. If you click **Change owner**, select the new owner from the list.

To deselect the alerts, click **Clear**.

## Alert details

> Click an alert to view more information about it in a sidebar, including:

Reviewer: ```suggestion
To view more information about alert, click on the alert's row to open the sidebar, which displays:
```

> - Error message for failing alerts.

Reviewer: ```suggestion
- Error messages for failing alerts.
```

- Link to the question that triggers the alert.

> - **Check history**: The results of the alert's recent checks.
> - **Send history**: The results of the alert's recent send attempts.

Reviewer: ```suggestion
- **Check history**: The status of the alert's recent checks.
- **Send history**: The status of the alert's recent send attempts.
```
Results would mean the payload.
